### ISSUE TITLE
BQ views Profiler; Errors on system metric computation

### ISSUE THREAD

**Issue by TeddyCr** [open-metadata/OpenMetadata#17681]
**Affected module**
openmetadata-ingestion

**Describe the bug**
View type in BQ throws the below error when computing system metrics
```
[2024-09-02T09:42:18.439+0000] {status.py:77} DEBUG - Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/metadata/profiler/interface/sqlalchemy/profiler_interface.py", line 382, in _compute_system_metrics
    rows = metrics().sql(session, conn_config=self.service_connection_config)
  File "/home/airflow/.local/lib/python3.10/site-packages/metadata/profiler/metrics/system/system.py", line 462, in sql
    system_metrics = get_system_metrics_for_dialect(
  File "/home/airflow/.local/lib/python3.10/site-packages/metadata/utils/dispatch.py", line 99, in wrapper
    return dispatch(str(args[0]))(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/metadata/profiler/metrics/system/system.py", line 140, in _
    jobs = [
  File "/home/airflow/.local/lib/python3.10/site-packages/metadata/profiler/metrics/system/system.py", line 141, in <listcomp>
    BigQueryQueryResult(
  File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for BigQueryQueryResult
dml_statistics
  none is not an allowed value (type=type_error.none.not_allowed)
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/metadata/profiler/interface/sqlalchemy/profiler_interface.py", line 446, in compute_metrics_in_thread
    row = self._get_metric_fn[metric_func.metric_type.value](
  File "/home/airflow/.local/lib/python3.10/site-packages/metadata/profiler/interface/sqlalchemy/profiler_interface.py", line 386, in _compute_system_metrics
    handle_query_exception(msg, exc, session)
  File "/home/airflow/.local/lib/python3.10/site-packages/metadata/profiler/interface/sqlalchemy/profiler_interface.py", line 62, in handle_query_exception
    raise RuntimeError(exc)
RuntimeError: 1 validation error for BigQueryQueryResult
dml_statistics
  none is not an allowed value (type=type_error.none.not_allowed)
```

**To Reproduce**
Run profiler for a BigQuery view -- system metric should throw the above error.

**Version:**
 - Python version:
 - OpenMetadata version: [e.g. 0.8]
 - OpenMetadata Ingestion package version: [e.g. `openmetadata-ingestion[docker]==XYZ`]

**Additional context**

**Comment by Cir02** [comment]
Hi, any updates on this issue?

thx!

**Comment by ayush-shah** [comment]
Hello @Cir02 this has been resolved in the upcoming 1.6.0 release, the above should not be reproducible post the upgrade once it is out, it should be published by late November or early december.
Fixed with #18322 
Thanks

**Comment by Cir02** [comment]
Thanks [ayush-shah](https://github.com/ayush-shah) !
